tools/plist_to_html/tests/libtest/env.py

```diff
@@ -0,0 +1,30 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+"""
+Test environment setup and configuration helpers.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+
+import os
+import tempfile
+
+
+def test_proj_root():
+    return os.path.abspath(os.environ['TEST_PROJ'])
+
+
+def get_workspace(test_id='test'):
+    tmp_dir = os.path.join(os.environ['REPO_ROOT'], 'build')
```

Comment:
Shouldn't we handle the newly introduced `CC_TEST_WORKSPACE_ROOT` env var here too?

Reply:
I've created a separate env var for this: `PLIST_TO_HTML_TEST_WORKSPACE_ROOT`